acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +82,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
-          step "assert no redacted data in syslog" do
-            #sigh.  gee, thanks for the help, beaker!
-            result = dump_puppet_log(agent)
```

Comment:
The return value from `dump_puppet_log` is the ![logger](https://github.com/puppetlabs/beaker/blob/3.1.0/lib/beaker/dsl/helpers/puppet_helpers.rb#L286), which can have other data from the Beaker test run in it, and not strictly just the syslog contents.